include/glow/Runtime/HostManager/HostManager.h

```diff
@@ -50,15 +51,41 @@ class HostManager final {
     /// safety.
     std::atomic<size_t> refcount;
   };
+  /// Container for inference requests waiting in the queue.
+  struct InferRequest {
+    std::string networkName;
+    std::unique_ptr<ExecutionContext> context;
+    ResultCBTy callback;
+    uint64_t priority;
```

Comment:
is priority going to be provided at onnxifi level?

Reply:
yes the plumbing for that still needs to be added. It should be straight forward so I'll add it to this PR.